src/app/features/charts/components/score-chart/score-chart.component.ts

```diff
@@ -91,6 +88,11 @@ export class ScoreChartComponent extends BaseChartComponent {
     });
   }
 
+  getMinDiff(count: number, distribution: any[]) {
+    const min = Math.min(...distribution.map(d => count - d.value).filter(v => v > 0));
```

Comment:
This is a really dense line to read.  Please consider breaking this logic into multiple lines.

Reply:
I've updated the method and added some documentation to make it easier to follow